src/qililab/qprogram/structured_program.py

```diff
@@ -250,10 +262,22 @@ def _float_variable(label: str, domain: Domain = Domain.Scalar) -> FloatVariable
 
         if domain == Domain.Time:
             return _int_variable(label, domain)
-        if domain in [Domain.Frequency, Domain.Phase, Domain.Voltage]:
+        if domain in [Domain.Frequency, Domain.Phase, Domain.Voltage, Domain.Flux]:
             return _float_variable(label, domain)
         raise NotImplementedError
 
+    def crosstalk(self, crosstalk: CrosstalkMatrix | None = None, calibration: Calibration | None = None):
+        """Declare Crosstalk matrix and offsets
+
+        Args:
+            crosstalk (CrosstalkMatrix): Crosstalk class including all necessary information
+        """
+        if calibration:
+            crosstalk = calibration.crosstalk_matrix
+        if not crosstalk:
+            raise ValueError("crosstalk must have either Calibration with crosstalk or CrosstalkMatrix")
+        self._crosstalk = crosstalk
```

Comment:
Why do we allow `Calibration` as a parameter in a method that sets the crosstalk? I think it is clearer to allow only `CrosstalkMatrix` and if the user has a `Calibration` instead just access it by `calibration.crosstalk_matrix`